### ISSUE TITLE
Add Netlify build plugins

### ISSUE THREAD

**Issue by seancdavis** [seancdavis/twenty-ninety#3]
Add support for the following plugins:

- a11y, but don't fail the build, only warn
- inline-critical-css
- subfont

Add a custom font to show that the font processing works as expected.
